api/v1/tests.py

```diff
@@ -7,25 +7,32 @@
 from fixtureless.factory import create
 
 from recipes.models import Recipe
+from recipes.serializers import RecipeSerializer
 
 
 class TestRecipeApi(APITestCase):
     def setUp(self):
         self.user = get_user_model().objects.create(username='test_user')
         self.recipe = create(Recipe, {'title': 'testing'})
+        self.recipe_list_url = reverse_lazy('api:v1:recipe-list')
+        self.recipe_detail_url = reverse_lazy('api:v1:recipe-detail', kwargs={'pk': self.recipe.pk})
 
     def test_recipe_list(self):
         """
         Verify GET returns Recipe list
         """
-        response = self.client.get(reverse_lazy('api:v1:recipe-list'))
+        response = self.client.get(self.recipe_list_url)
+        recipes = Recipe.objects.all()
+        serializer = RecipeSerializer(recipes, many=True)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Recipe.objects.count(), 1)
```

Comment:
you're already caching `Recipe.objects.all()` under the variable `recipes`. You can then just use this here instead of doing `Recipe.objects.count()`. Just do `recipes.count()`